eras/conway/test-suite/cddl-files/conway.cddl

```diff
@@ -67,15 +67,15 @@ transaction_body =
   , ? 16 : transaction_output       ; collateral return
   , ? 17 : coin                     ; total collateral
   , ? 18 : set<transaction_input>   ; reference inputs
-  , ? 19 : [* voting_procedure]     ; New; Voting procedures
+  , ? 19 : voting_procedures        ; New; Voting procedures
   , ? 20 : [* proposal_procedure]   ; New; Proposal procedures
   }
 
+voting_procedures = { + voter => { + governance_action_id => voting_procedure } }
```

Comment:
It looks to me like the actual decoder would happily decode an empty outer map, so the CDDL might be too strict here
```suggestion
voting_procedures = { voter => { + governance_action_id => voting_procedure } }
```

Reply:
No, empty Map in Haskell means that the field is missing in CBOR.

See decoder for the `mint` field. It is the exact same approach.